src/peaks/getPeaks.js

```diff
@@ -10,21 +10,19 @@ export function getPeaks(chromatogram, options = {}) {
 
   const series = chromatogram.getSeries(seriesName).data;
   const times = chromatogram.getTimes();
+  const dataXY = { x: times, y: series };
   // first peak selection
-  let peakList = gsd(
-    { x: times, y: series },
-    {
-      noiseLevel: 0,
-      realTopDetection: false,
-      smoothY: true,
-      sgOptions: { windowSize: 5, polynomial: 2 },
-      heightFactor: 2,
-    },
-  );
+  let peakList = gsd(dataXY, {
+    noiseLevel: 0,
+    realTopDetection: false,
+    smoothY: true,
+    sgOptions: { windowSize: 5, polynomial: 2 },
+  });
+
   // filter height by factor
   let medianHeight = median(series);
 
-  peakList = peakList.filter((val) => val.height > medianHeight * heightFilter);
+  peakList = peakList.filter((val) => val.y > medianHeight * heightFilter);
```

Comment:
In the previous versions of ml-gsd, if `heightFactor` was passed, it added a "height" property, which has nothing to do with `y`. Need to investigate what it was.

Reply:
![image](https://user-images.githubusercontent.com/2352663/156771256-8c85ea3e-e6fe-4092-a13d-e408ac295334.png)